src/components/SavedProjectPlan/StandardCard.jsx

```diff
@@ -0,0 +1,42 @@
+import {
+  Box,
+  Card,
+  CardBody,
+  CardHeader,
+  Flex,
+  Heading,
+  Image,
+} from "@chakra-ui/react";
+
+const StandardCard = ({ title, imageUrl, criteria }) => {
+  const [firstImage, secondImage] = imageUrl;
+
+  return (
+    <Card flex="1">
+      <CardHeader>
+        <Heading size="lg">{title}</Heading>
```

Comment:
The actual design has no boxShadow property. Try removing that since chakra ui card component by default comes with a box shadow.